service/pokemon_service.go

```diff
@@ -0,0 +1,25 @@
+package service
+
+import (
+	"WizelineApi/domain/model"
+	"errors"
+)
+
+//Servicio utilizado para buscar por ID dentro de un slice del modelod e Pokemon
+func GetPokemonById(pokemons []model.Pokemon, id int) (model.Pokemon, error){
+	t_pkm := model.Pokemon{}
+
+	for _,pk := range pokemons{
+		if pk.ID == id {
+			t_pkm = pk
+			break
+		}
+	}
+
+	if t_pkm.Name == "" {
+		return t_pkm, errors.New("Couldnt find pokemon")
+	}else{
+		return t_pkm, nil
+	}
```

Comment:
Pro-tip. You could remove the else and the code is going to behave the same way.
```suggestion
	if t_pkm.Name == "" {
		return t_pkm, errors.New("Couldnt find pokemon")
	}
	
	return t_pkm, nil
```